bin/minifollowups/pycbc_sngl_minifollowup

```diff
@@ -91,6 +96,7 @@ parser.add_argument('--output-file')
 parser.add_argument('--tags', nargs='+', default=[])
 wf.add_workflow_command_line_group(parser)
 args = parser.parse_args()
+args.statistic_files = sum(args.statistic_files, [])
```

Comment:
What does this do?

Reply:
The reason I ask, is this looks like it should be redundant. The default is already to '[]', no, so does this line have any effect?

Reply:
This flattens the list (and matches what you yourself coded in the coinc code). Because statistic files can come from within the workflow (FIT_BY_PARAMS) and externally (the PHASE_TIME input file), this option needs to be given twice and the two lists are appended. So `statistic-files` becomes a list of list and this flattens it with `[]` being the default if no entries are given.

But I just copied this from your code, and it *is* needed.

Reply:
Ok, I guess that makes sense. I had forgotten about this use.

Reply:
sounds like something you'd want to put a comment in to explain ..?

Reply:
Comment added